assets/src/components/routePicker.tsx

```diff
@@ -60,28 +54,43 @@ const RoutesList = ({
   const dispatch = useContext(DispatchContext)
 
   return (
-    <ul>
+    <ul className="m-route-picker__route-list">
       {routes.map(route => (
         <li key={route.id}>
-          {selectedRouteIds.includes(route.id) ? (
-            <button
-              className="m-route-picker__route-list-button--selected"
-              onClick={() => dispatch(deselectRoute(route.id))}
-            >
-              *{route.id}
-            </button>
-          ) : (
-            <button
-              className="m-route-picker__route-list-button--deselected"
-              onClick={() => dispatch(selectRoute(route.id))}
-            >
-              {route.id}
-            </button>
-          )}
+          <RouteListButton
+            route={route}
+            isSelected={selectedRouteIds.includes(route.id)}
+            selectHandler={() => dispatch(selectRoute(route.id))}
+            deselectHandler={() => dispatch(deselectRoute(route.id))}
+          />
         </li>
       ))}
     </ul>
   )
 }
 
+const RouteListButton = ({
+  route,
+  isSelected,
+  selectHandler,
+  deselectHandler,
+}: {
+  route: Route
+  isSelected: boolean
+  selectHandler: () => void
+  deselectHandler: () => void
+}) => {
+  const classModifier = isSelected ? "selected" : "deselected"
```

Comment:
Unless you decide that specifying the background color with the `--deselected` modifier is good, there's no css referencing `--deselected` and it could be removed.